hoot-core/src/main/cpp/hoot/core/ops/GeometryModifierOp.cpp

```diff
@@ -104,29 +104,29 @@ QList<GeometryModifierActionDesc> GeometryModifierOp::_readJsonRules()
     // check command availability
     foreach (boost::shared_ptr<GeometryModifierAction> pAction, _actions)
     {
-      if( pAction->getCommandName() == actionDesc.command )
+      if (pAction->getCommandName() == actionDesc.command)
```

Comment:
Lots of these changes per the coding standard.  `if`, `else if`, `for`, `while`, etc. all should have a space after them preceding the opening parenthesis and not after the parenthesis.